### DISCUSSION TITLE
Update coefficients in an exsiting model easily

### DISCUSSION THREAD

**Discussion by Zhanwei-Liu** [metab0t/PyOptInterface#10] [Q&A] [answered]
Hi,

Does `PyOptInterface` have a function akin to `pyomo`'s [`Param`](https://pyomo.readthedocs.io/en/stable/pyomo_modeling_components/Parameters.html) (`mutable=Ture`) that allows for easy updating of model coefficients? Alternatively, does anyone have suggestions on how to update the coefficients in a model using `PyOptInterface`?

Best Regards,

**Comment by metab0t** [comment, marked as answer]
No, full support for parameter is a complex issue. Even JuMP.jl does not have a complete solution for this problem.

However, for linear constraints, we provide methods to modify its coefficients and right-hand side directly as documented in https://metab0t.github.io/PyOptInterface/constraint.html#modify-constraint
